testament/tester.nim

```diff
@@ -613,32 +645,37 @@ proc main() =
       if kind == pcDir and cat notin ["testdata", "nimcache"]:
         cats.add cat
     cats.add AdditionalCategories
+    if useMegatest: cats.add MegaTestCat
 
     var cmds: seq[string]
     for cat in cats:
-      cmds.add(myself & " pcat " & quoteShell(cat) & rest)
+      let runtype = if useMegatest: " pcat " else: " cat "
```

Comment:
``pcat`` is "run testament in parallel and has nothing to do with ``useMegatest``.

Reply:
oh, that's funny because like the comment says, the only difference between `pcat` and `cat` is that `cat` skips joined tests and `pcat` doesn't, so far as I can tell at least - the megatest implementation is quite.. confusing.

what would be a better way of disabling megatests?

Reply:
My bad, you're right and the source code comment is right.